src/common/libflux/plugin.c

```diff
@@ -34,6 +36,21 @@
 #define UUID_STR_LEN 37     // defined in later libuuid headers
 #endif
 
+static int use_deepbind = 1;
+static pthread_once_t deepbind_once = PTHREAD_ONCE_INIT;
+static void init_use_deepbind () {
+    const char *var = getenv ("FLUX_LOAD_WITH_DEEPBIND");
+    if (!var) {
+        // default is to allow the flag
+        return;
+    }
+    use_deepbind = strtol (var, NULL, 10);
+}
+
+int plugin_deepbind () {
+    pthread_once(&deepbind_once, init_use_deepbind);
```

Comment:
Style: open brace for function on new line, space before open paren in pthread_once